functions.photon.php

```diff
@@ -99,6 +118,7 @@ function jetpack_photon_url( $image_url, $args = array(), $scheme = null ) {
 	if (
```

Comment:
Should probably add the reason for the `|| $is_wpcom_image_with_safe_args` in this comment, otherwise it's presence seems bit out of place.